src/generic/treectlg.cpp

```diff
@@ -2662,12 +2662,13 @@ void wxGenericTreeCtrl::PaintItem(wxGenericTreeItem *item, wxDC& dc)
     {
         dc.SetClippingRegion(item->GetX() + state_w, item->GetY(),
                              image_w, total_h);
-        GetImageList()->Draw( image, dc,
-                                 item->GetX() + state_w,
-                                 item->GetY() +
-                                    (total_h > image_h ? (total_h-image_h)/2
-                                                       : 0),
-                                 wxIMAGELIST_DRAW_TRANSPARENT );
+
+        const wxCoord x = item->GetX() + state_w;
+        const wxCoord y = item->GetY() + (total_h > image_h ? (total_h-image_h)/2 : 0);
+
+        auto &img = GetImages().at(image);
```

Comment:
We really need to use `GetBitmapBundle()` here, otherwise it will crash, AFAICS, if the existing code uses `wxImageList`-based API and not `wxBitmapBundle`-based one for adding the images. I.e.
```suggestion
        auto &img = GetBitmapBundle(image);
```